### DISCUSSION TITLE
DAIKIN MOD. ATXN25NV1B9

### DISCUSSION THREAD

**Discussion by grbasta** [revk/xESP32-Faikout#79] [Q&A]
Sembrerebbe che questo modello non ha a disposizione nessun adattatore wifi disponibile, ma comunque ha il connettore S21 ma la dicitura sulla scheda come da foto allegata della scheda si chiama cn_wired.

![WhatsApp Image 2023-07-25 at 14 27 33](https://github.com/revk/ESP32-Faikin/assets/46572022/18e16842-6b70-4966-b3ad-745255dca6fa)

**Comment by grbasta** [comment]
It would seem that this model does not have any wifi adapter available, but it still has the S21 connector but the wording on the board as per the attached photo of the board is called cn_wired.

**Reply by 44616E** [reply]
Different protocol, but they are working to add it (work in progress)
